atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -103,8 +105,12 @@ def run(self):
             else:
                 err_msg = ('Could not get koji build info for parent image {}. '
                            'Was this image built in OSBS?'.format(img.to_str()))
-                self.log.error(err_msg)
-                raise RuntimeError(err_msg)
+                if get_allow_external_base_image(self.workflow, fallback=False):
+                    self.log.warning(err_msg)
+                    del self._parent_builds[img]
```

Comment:
I don't think we want to remove it

in koji metadata : image->prebuild_build_id   and image->parent_image_builds
should still be something, None for build_id and just key with imagename used with {} as value for parent_image_builds
so it is clear that really doesn't have any koji build